packages/contracts-bedrock/snapshots/abi/OptimismPortal2.json

```diff
@@ -19,6 +19,19 @@
     "stateMutability": "payable",
     "type": "receive"
   },
+  {
+    "inputs": [],
+    "name": "balance",
```

Comment:
Verified the ABI diff as well. Aside from just looking at the ABI diff in this PR I also compared the OptimismPortal and OptimismPortal2 ABIs—no missing or extra gas token methods found ✅